### PR TITLE
Add PR template

### PR DESCRIPTION
**Pull request by osfanbuff63**
Adds a PR template with a few notes about how we accept PRs.

(please squash and merge, had to rebase over my changes which caused merge conflicts even though I don't even remember what the last change I made on this clone was haha)

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<!--
+Thanks for submitting a pull request!
+
+If you are adding or updating translations, here's a helper guide: https://fabulously-optimized.gitbook.io/modpack/readme/language-support
+If this is a translation update or addition for Mod Menu Helper, make sure it will fit in Minecraft either by testing it using this modpack and/or this online tool: https://fabulously-optimized.github.io/Mod-Menu-Helper-Size-Checker/
+
+Keep in mind that we **do not** accept mod or configuration changes using pull requests, please create an issue here instead: https://github.com/Fabulously-Optimized/fabulously-optimized/issues/new/choose.
```
Comment (Kichura) [inline .github/pull_request_template.md:7]: Use `https://github.com/Fabulously-Optimized/fabulously-optimized/issues/new/choose/` as github for some reason will assume the period belongs inside the URL which isn't what we want.

**Review by Kichura — CHANGES_REQUESTED** [review]

File: .github/pull_request_template.md (hunk 1 above)
Comment (osfanbuff63) [reply .github/pull_request_template.md:7]: in vscode this didn't count as part of the url

**Review by Kichura — APPROVED** [review]
This is good, well done.

**Comment by Madis0** [comment]
Could you do it as [form style](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema)?

Essentially one "label" of text and one big textbox for contents

**Comment by osfanbuff63** [comment]
> Could you do it as [form style](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema)?
> 
> Essentially one "markdown" of text and one big textarea for contents

Can try; I've never tried that with PR templates though

**Comment by osfanbuff63** [comment]
This doesn't seem to work, or at least how Issues do...
image

**Comment by Madis0** [comment]
> Issue forms are not supported for pull requests. 

Oops, you're right. Revert and fix open issues then.

File: .github/pull_request_template.md (hunk 1 above)
Comment (osfanbuff63) [reply .github/pull_request_template.md]: ~~forgot to resolve this :ioa:~~

**Pull request merged**